host_vars/dev.gvl.org.au.yml

```diff
@@ -32,7 +32,7 @@ certbot_domains:
   - "genome.{{ hostname }}"
 certbot_dns_provider: cloudflare
 certbot_dns_credentials:
-  api_token: "{{ vault_dns_cloudflare_api_token }}"
+  api_token: "{{ vault_dns_cloudflare_gvl_api_token }}"
```

Comment:
does this token need to be used for dev-queue too?

Reply:
I don't think so - it should only be needed where `certbot_dns_credentials` is defined, which for the dev vms is only `host_vars/dev.gvl.org.au.yml`